BUILD

```diff
@@ -58,3 +58,13 @@ load("@bazel_gazelle//:def.bzl", "gazelle")
 # gazelle:proto package
 # gazelle:proto_group go_package
 gazelle(name = "gazelle")
+
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+
+# TODO: This is only here in order to workaround a bug in the way bazel resolves
```

Comment:
can you link to an issue number, if it exists? (or some way for somebody to check whether the bug is fixed)

Reply:
I'm currently talking to someone on slack about how to fix this, don't have a bug yet but should soon, need to do some more testing a repros first.